telco-hub/configuration/reference-crs-kube-compare/optional/lso/lsoLocalVolume.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: {{ .metadata.name }}
+  namespace: "openshift-local-storage"
+spec:
+  logLevel: Normal
+  managementState: Managed
+  {{- if .spec.nodeSelector }}
+  nodeSelector:
+    {{- .spec.nodeSelector | toYaml | nindent 4 }}
+  {{- end }}
+  storageClassDevices:
+    {{- range .spec.storageClassDevices }}
+    - storageClassName: {{ .storageClassName }}
+      forceWipeDevicesAndDestroyAllData: true
+      volumeMode: Block
```

Comment:
Block or Filesystem is ok.

Reply:
Done. Both options allowed in the template.